### ISSUE TITLE
xcdatamodeld not handled properly

### ISSUE THREAD

**Issue by AliSoftware** [liftoffcli/liftoff#192]
Not sure if it's a liftoff issue or a [CocoaPods/Xcodeproj](https://github.com/CocoaPods/Xcodeproj) one, but if I add a `xcdatamodeld` file to my template, it gets added as a regular file (`PBXFileReference`) in the generated Xcode project, instead of a special entry for `xcdatamodeld` files (`isa = XCVersionGroup` & `versionGroupType = wrapper.xcdatamodel`).

As a result, Xcode does not recognize it as an actual CoreData model.

Removing the `xcdatamodeld` reference from the Xcode project and adding it again manually solves the problem.

---
## Steps to reproduce
- Create a `Model.xcdatamodeld` file using Xcode
- Save it in `~/liftoff/templates/Model.xcdatamodeld`
- Add an entry for it in your `~/.liftoffrc` file
- Generate a new project using `liftoff` command line
## Actual Result

The `Model.xcdatamodel` is present in the Project Navigator but when selecting it, it does not recognize it as an CoreData model and the data model does not get displayed in the main editor, because it has been added as a `PBXFileReference`.
## Expected Result

The `Model.xcdatamodel` should have been added as a `XCVersionGroup` entry with `versionGroupType = wrapper.xcdatamodel` so that Xcode recognizes it and display its content when selected in the Project Navigator.

---

```
$ liftoff --version
Version: 1.4.1
$ xcproj --version
0.1.2
```

**Comment by adamhrz** [comment]
@AliSoftware @gfontenot @Ashton-W Submitted a PR for this in CocoaPods/Xcodeproj#291 - tests are failing but I'm not familiar enough with Ruby tests to dig in. Bottom line xcodeproj is fine if your datamodel lives in your project root, otherwise it fails to create the children and current version since path.exists? fails

**Comment by Eldorado234** [comment]
Should be fixed in Xcodeproj version `1.3.3` and higher.
